docs/en_US/tasks/configure-emqx-tls.md

```diff
@@ -123,6 +61,8 @@ There are many types of Volumes. For the description of Volumes, please refer to
             cacertfile = "/mounted/cert/ca.crt"
             certfile = "/mounted/cert/tls.crt"
             keyfile = "/mounted/cert/tls.key"
+            gc_after_handshake = true
+            hibernate_after = 5s
```

Comment:
Why need this? have any documents ?

Reply:
https://www.emqx.io/docs/zh/v5/admin/cfg.html#broker-listener-ssl-opts
https://www.emqx.io/docs/en/v4.4/changes/changes-4.4.html#v4-4-11
Improve performance